cc

```diff
@@ -98,6 +107,12 @@ try:
     # Prepend and append new items.
     argv = link.get("prepend", []) + argv + link.get("append", [])
 
+    for name in link.get("insert-target-rlib", []):
```

Comment:
```suggestion
    for name in link.get("insert-rlib", []):
```

Is this an improvement or not? I'm on the fence.

Reply:
```suggestion
    for name in link.get("append-rlib", []):
```

Using "append" matches the existing language.

Reply:
hmm, I wanted to distinguish normal `rlib` from the `target-rlib`s ..